packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -316,6 +317,38 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
       const abortController = new AbortController();
 
       if (isOnline) {
+        if (visitToEdit?.uuid) {
+          const existingAttributes = visitToEdit.attributes.map((attr) => attr.attributeType.uuid) || [];
+          const updatedAttributes = [];
+          const addedAttributes = [];
+          payload.attributes.forEach((attribute) => {
+            if (existingAttributes.includes(attribute.attributeType)) {
+              const attributeUuid = visitToEdit.attributes.filter(
+                (attr) => attr.attributeType.uuid === attribute.attributeType,
```

Comment:
Fix: `attribute.attributeType.uuid`

Reply:
`attribute` is already used in the scope @vasharma05